apps/book/filters.py

```diff
@@ -12,6 +12,7 @@ class BookFilter(django_filters.FilterSet):
     is_added_in_wishlist = django_filters.rest_framework.BooleanFilter(
         method='filter_is_added_in_wishlist', initial=False
     )
+    grade = StringListFilter(method='filter_grade')
```

Comment:
Use an Enum filter instead of a string.

For eg: https://github.com/the-deep/server/blob/develop/apps/lead/filter_set.py#L317